packages/wallet/HARDWARE_TESTING.md

```diff
@@ -0,0 +1,110 @@
+# Hardware Testing Environment
```

Comment:
The new `./scripts/*` are not referenced anywhere in the docs, please either reference or delete

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1651/commits/918568cfcc56b8f21812e348bb14908fefe3027e